utils/collections/getIssueMetadata.ts

```diff
@@ -0,0 +1,54 @@
+import { Collection } from 'types';
+
+const getIssuePrintIssn = (collection) => {
+	if (collection.metadata.printIssn) {
+		return collection.metadata.printIssn;
+	}
+	return collection.metadata && collection.metadata.printIssn;
+};
```

Comment:
I love the use of small functions here, but if I understand their purpose, we might be able to simplify their implementation a bit.

Starting with the return statement, it looks like we're assuming that a collection may not have a `metadata` property. If that's the case, the first line of the function (line 4) is cause for concern, because if `metadata` is undefined, we'll have a `TypeError: Cannot read property of undefined` before we ever get to line 7.

These functions really just access a deeply nested property, returning `undefined` if an object in the chain is also undefined. The easiest way to simplify while preserving the original intent might be to use the [Optional chaining operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) like so:

```ts
const getIssuePrintIssn = (collection) => collection.metadata?.printIssn;
```